encoding/mkv.md

```diff
@@ -0,0 +1,505 @@
+1. Extracting audio tracks from BDMV .m2ts files
+1. Trimming audio to match our VapourSynth video trims
+1. Creating mkv files from individual tracks
+1. Creating chapters for said files
+1. Creating virtual timelines with ordered chapters
+1. Using multiple audio codecs in the same 'apparent file'
+1. Using editions to change between NCOP and OP
+1. Creating a playall file with ordered chapters
+
+[toolnix]: https://mkvtoolnix.download/downloads.html
+[FFmpeg]: https://ffmpeg.zeranoe.com/builds/
+[mk-playback]: https://github.com/hubblec4/Matroska-Playback/blob/e48149b500ce80c38ebc07ccf8f74f8af45e89b2/src/PlayerOverview.md
+[MPC-HC]: https://github.com/clsid2/mpc-hc/releases
+[mpv]: https://mpv.io/installation/
+[acsuite.py]: https://github.com/OrangeChannel/acsuite
+
+
+### Starting with a clean work environment
+
+The following steps should be done from within a folder containing
+all of the files (episodes) re-named to some simple format like `ep##.m2ts`,
+as having multiple `00004.m2ts` files won't be helpful.
+The example scripts provided will assume this workflow.
+
+
+## Extracting the audio
+
+If the source files you are working with are not .m2ts,
+re-encoding your audio into PCM / WAV is *still* recommended
+as slicing some lossless or lossy codecs is not precise.
+<!-- TODO: which ones? -->
+If you are working with a BDMV source,
+it is likely you will need a more complex method
+to extract an audio track from the transport stream files.
+This can be done with FFmpeg or [tsMuxeR][].
+Using FFmpeg we can extract only the audio stream 
+using the following command[^1]:
+
+```sh
+ffmpeg -i ep01.m2ts -map a:0 ep01.wav
+```
+
+If the desired audio track is not the first,
+`-map a:#` should be changed to the track # shown by `ffprobe`.
+
+*If you are following along with the next few steps,
+you don't need to extract the audio just yet.
+The audio-cutting script will apply this FFmpeg command to the files
+automatically.*
+
+[tsMuxeR]: https://www.videohelp.com/software/tsMuxeR
+
+
+## Starting with trimming the video
+
+Provided are examples of trims that may need to be made
+to cut the OP/ED out of an episode,
+and possibly any other unwanted frame-ranges
+(some blu-ray streams have a silent black screen at the end,
+which should be cut out).
+
+Starting with the BDMVs,
+our first step is to trim the main episodes out.
+This requires knowing the frame ranges, shown below.
+Second, we will crop out the NCOP/NCED files
+as they also have unwanted frames at the end.
+
+The first time the OP appears,
+we will also want to note this frame range
+ and on what episode it occurs,
+as we will only need to encode the OP once.
+Same goes for the ED.
+
+```py
+import vapoursynth as vs
+core = vs.core
+
+bdmv01 = core.lsmas.LWLibavSource('ep01.m2ts')
+bdmv02 = core.lsmas.LWLibavSource('ep02.m2ts')
+...
+bdmv13 = core.lsmas.LWLibavSource('ep13.m2ts')
+bdmvNCED = core.lsmas.LWLibavSource('nced.m2ts')
+bdmvNCOP = core.lsmas.LWLibavSource('ncop.m2ts')
```

Comment:
I personally find it pretty weird to do all of this in a single file instead of the normal per-episode encode script and it looks like you need to convert these trims into separate function calls for acsuite later anyway.

Regardless, the way I currently understand it it would lead to having to keep two scripts in sync regarding the trims you make and might adjust later.